app/components/candidate_interface/degree_new_review_component.rb

```diff
@@ -271,15 +271,19 @@ def generate_action(degree:, attribute: '')
     def formatted_degree_type(degree)
       return if degree.qualification_type.nil?
 
-      reference_data = DfE::ReferenceData::Degrees::TYPES.some_by_field(:name).keys.select { |type| degree.qualification_type.include?(type) }
-      if reference_data.present?
-        degree.qualification_type.split.first
+      if degree.qualification_level.present?
+        DegreeWizard::QUALIFICATION_LEVEL[degree.qualification_level]
+      else
+        reference_data = DfE::ReferenceData::Degrees::TYPES.some_by_field(:name).keys.select { |type| degree.qualification_type.include?(type) }
+        degree.qualification_type.split.first if reference_data.present?
```

Comment:
if we're retaining this then we may have a mismatch. E.g this will render Bachelor for type of degree row instead of Bachelor degree. The key in the row will be Type of Bachelor degree, instead of Type of bachelor degree. To replicate, type in Bachelor of Arts in the another qualification text field on degree level page. Small nitpick tho.

Reply:
I see. Is this is a blocker to merge?

Reply:
Fixed!